tests/src/JIT/opt/InstructionCombining/DivToMul.csproj

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The project files were updated recently, could you please copy one of new csproj here instead of this one in an old format?